tests/formats/pdb.cpp

```diff
@@ -246,51 +287,34 @@ TEST_CASE("Read files in PDB format") {
         REQUIRE(frame.size() == 2223);
     }
 
-    SECTION("Read file generated by Crystal Maker") {
+    SECTION("File generated by Crystal Maker") {
         auto file = Trajectory("data/pdb/crystal-maker.pdb");
         REQUIRE(file.nsteps() == 1);
 
         auto frame = file.read();
         REQUIRE(frame.size() == 8);
     }
 
-    SECTION("Read Protein Residues") {
-        auto frame = Trajectory("data/pdb/hemo.pdb").read();
-        auto topology = frame.topology();
-
-        CHECK(!topology.are_linked(topology.residue(2), topology.residue(3)));
-        CHECK( topology.are_linked(topology.residue(3), topology.residue(4)));
-        CHECK(!topology.are_linked(topology.residue(3), topology.residue(5)));
-        CHECK(topology.bonds().size() == 482);
+    SECTION("Short CRYST1 record") {
+        auto file = Trajectory("data/pdb/short-cryst1.pdb");
+        auto frame = file.read();
     }
 
-    SECTION("Read Nucleic Residues") {
-        auto frame = Trajectory("data/pdb/2hkb.pdb").read();
-        auto topology = frame.topology();
+    SECTION("Short ATOM record") {
+        auto file = Trajectory("data/pdb/short-atom.pdb");
```

Comment:
Does this supersede the below test?

Reply:
No, this is a new test for the bug fixed by this PR.